Noh doc postprocessing sw (#33)
ices-eg/wg_WGFAST: SONAR-netCDF4/docs/crr341.adoc

The convention version number will always be included in the title of the document. The document version number will always be found in <<_revision_history>>.

== The recommended use of the SONAR-netCDF4 convention in post processing software

> There are several processing steps that are common among the different postprocessing tools used by the community, and there is a growing need to use the different tools interchangeably. This chapter describes a few standard processing steps and outlines how the standard is intended to be used to facilitate the interchangeability. Note that the steps may be different between software solutions, and the convention only recommend how to store the data for the steps that are most common and we do not address how and which order to process your data.

Reviewer: ```suggestion
There are several processing steps that are common among the different postprocessing tools used by the community, and there is a growing need to use the different tools interchangeably. This chapter describes a few standard processing steps and outlines how the standard is intended to be used to facilitate this interchangeability. Note that the steps may be different between tools, and that the convention only recommends how to store the data for the steps that are most common. We do not address how and in which order to process your data.
```

> There are several processing steps that are common among the different postprocessing tools used by the community, and there is a growing need to use the different tools interchangeably. This chapter describes a few standard processing steps and outlines how the standard is intended to be used to facilitate the interchangeability. Note that the steps may be different between software solutions, and the convention only recommend how to store the data for the steps that are most common and we do not address how and which order to process your data.

Reviewer: ```suggestion
There are several processing steps that are common among the different postprocessing tools used by the community, and there is a growing need to use the different tools interchangeably. This chapter describes a few standard processing steps and outlines how the convention is intended to be used to facilitate the interchangeability. Note that the steps may be different between tools, and the convention only recommends how to store the data for the steps that are most common. We do not address how and in which order to process your data.
```

> Exchange of information between different parts in the processing chain is becoming increasingly important. This is particularly relevant for emerging deep learning and machine learning frameworks, that has hardware requirements that often are different than the computers that run the traditional desktop applications. Processing tools to be run on autonomous platforms are also a need, which often require a different suite of software solutions, typically tailored to low power situations. 

Reviewer: ```suggestion
Exchange of information between different parts in the processing chain is becoming increasingly important. This is particularly relevant for emerging deep learning and machine learning frameworks, that have hardware requirements that are often different to the computers that are used for traditional desktop applications. Processing tools that run on autonomous platforms are also a need, which often require a different suite of software solutions, typically tailored to low power situations. 
```

> Typical steps in the postprocessing chain of marine sonar data consist of preprocessing and noise removal, bottom detection, classification, and integration (Figure 1). Here we outline the different groups in the convention that can support the input and output from these steps. If a processing software step can read the data from the standard, process the information and write back to the data using the convention, other software can pick it up and process it further. 

Reviewer: ```suggestion
Typical steps in the postprocessing chain of marine sonar data consist of preprocessing and noise removal, bottom detection, classification, and integration (<<postprocessing_data_flow_figure>>). Here we outline the different groups in the convention that can support the input and output from these steps. If a processing software step can read the data, process the information and write back to the data using the convention, other software can pick it up and process it further. 
```

A design principle of SONAR-netCDF4 has been to focus on describing the acoustic backscatter data, not the overall purpose and context of why the data were collected. Such broader metadata are better stored in separate metadata systems and schema (e.g. citenp:[iso2014,ices2016]).

> Figure 1. Typical steps in a postprocessing chaing for acoustical water column data. (TODO: Link these boxes to the groups in the convention and check if there are other needs)

Reviewer: ```suggestion
.Typical steps in a postprocessing chain for acoustical water column data. (TODO: Link these boxes to the groups in the convention and check if there are other needs)
[[postprocessing_data_flow_figure]]
image::postprocessingDataFlowFigure.svg[align="center"]

```
